### PR TITLE
Remove old version of JanDeDobbeleer.OhMyPosh

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/40941)

### CODE REVIEW

File: manifests/f/FengHe/Migi/2.15.1-670/FengHe.Migi.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://download.kirakuapp.com/FN_PC/Focus%20Note%20Setup%202.15.1-670.exe
+  InstallerSha256: 27581120F9EB74785040B041D026A1D05F35129C4E370DEA44BF3468EB3DF43B
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+  ProductCode: 812c6e4e-6ba1-5575-a2f8-dd470e948242
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: https://download.kirakuapp.com/FN_PC/Focus%20Note%20Setup%202.15.1-670.exe
```
Comment (OfficialEsco) [inline manifests/f/FengHe/Migi/2.15.1-670/FengHe.Migi.installer.yaml:21]: Still the wrong urls

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Comment by vedantmgoyal2009** [comment]
I will look at the issue, please don't close the PR.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/40941#issuecomment-1013708855

**Comment by vedantmgoyal2009** [comment]
Fixed the issue in automation.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**